src/client/utils/tracking/tracking.ts

```diff
@@ -110,9 +122,20 @@ export const adtraction = (
       adt.Tag.cpn = couponCode
     }
 
-    adt.Tag.tp = isBundle(offerData)
-      ? getComboAdractionProductValue(isYouth(offerData))
-      : adtractionProductMap[offerData.quotes[0].contractType]
+    const getTpTag = () => {
+      if (isBundle(offerData))
+        return getComboAdractionProductValue(isYouth(offerData))
+      const contractType = offerData.quotes[0].contractType
+      if (!(contractType in NOT_VALID_CONTRACTS)) {
+        return adtractionProductMap[
+          contractType as TypeOfContractExcludedUnused
+        ]
+      }
+      return null
+    }
+
+    const tpTag = getTpTag()
+    if (tpTag) adt.Tag.tp = tpTag
```

Comment:
Do we want to trigger the event if it's a dk quote?

Reply:
If we can remove every danish quote then we can skip to trigger it. Did not realise that the already existing danish types should also be put in the invalid type of contracts.
Then if that is the case we can just check if it's  danish quote and we can remove the list that I compare to. 
I thought that we wanted to keep the contracts that we had in the adtractionProductMap.
That makes it easier